client/__tests__/e2e/e2e.test.ts

```diff
@@ -90,12 +90,17 @@ const genesetDescriptionString = "fourth_gene_set: fourth description";
 const genesetToCheckForDescription = "fourth_gene_set";
 
 const data = datasets[DATASET];
+const dataTruncate = datasets[DATASET_TRUNCATE];
 
 const defaultBaseUrl = "d";
 const pageUrl = appUrlBase.includes("localhost")
   ? [appUrlBase, defaultBaseUrl, DATASET].join("/")
   : appUrlBase;
 
+  const pageUrlTruncate = appUrlBase.includes("localhost")
+  ? [appUrlBase, defaultBaseUrl, DATASET_TRUNCATE].join("/")
+  : appUrlBase;  
```

Comment:
I see it does it above as well, but why does this default to url base if it's not localhost?

Reply:
Good question